workflow/tests/base_results/results.csv

```diff
@@ -58,8 +58,8 @@ base-bldgtype-single-family-attached.xml,42.689,42.689,29.914,29.914,12.774,0.0,
+base-calctype-operational-residents-0-runperiod-1-month.xml,8.8432,8.8432,0.431,0.431,8.4122,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.1388,0.1034,0.0,0.0468,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.1421,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,8.4122,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,7.8746,0.0,0.0,0.0511,0.0,0.0,0.0,0.0,556.25,0.0,26.9062,0.0,0.0,0.6026,0.6425,0.0908,1.745,0.1094,-0.1538,0.0,0.0,2.2325,-0.0004,1.0008,0.0,0.0,0.0,1.7531,-0.1932,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,36000.0,24000.0,0.0,6.8,91.76,32302.0,8597.0,7508.0,0.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,17964.0,5216.0,7127.0,0.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-calctype-operational-residents-0.xml,41.462,41.462,6.723,6.723,34.739,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.573,2.868,0.484,0.578,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,34.739,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,32.54,8.342,0.0,0.622,0.0,0.0,0.0,0.0,544.2,1789.1,25.515,14.358,0.0,3.425,3.59,0.505,7.272,0.622,-2.267,0.0,0.0,7.973,-0.064,5.433,0.0,0.0,0.0,7.178,-1.415,0.0,0.226,-0.218,-0.017,3.317,0.037,10.433,0.0,0.0,-5.398,-0.059,-1.052,-2.28,0.0,0.0,2.085,1.426,0.0,0.0,0.0,0.0,36000.0,24000.0,0.0,6.8,91.76,32302.0,8597.0,7508.0,0.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,17964.0,5216.0,7127.0,0.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
These change slightly because we are now applying vacancy periods to clothes dryer exhaust schedules.